### PR TITLE
make demo pretty

### PR DESCRIPTION
**Pull request by notwaldorf**
/cc @cdata @morethanreal 

Looks like this:
screen shot 2015-05-20 at 4 19 31 pm

### CODE REVIEW

**Comment by morethanreal** [comment]
Nice, could we share the demo page styles in `paper-styles`, so they can be reused?

**Comment by notwaldorf** [comment]
Not sure it's super worth it. THe only thing relevant is the background and the section, and those have `custom-styles` so you'd have to remember to always import `demo.css` unlike a normal css. Is it worth it?

File: demo/index.html [hunk 1]
```diff
@@ -20,49 +20,97 @@
...
     <link rel="import" href="../paper-radio-group.html">
-
+    <link rel="import" href="../../paper-styles/paper-styles.html">
     <link rel="stylesheet" href="../../paper-styles/demo.css">
 
     <style is="custom-style">
-      paper-radio-group.red paper-radio-button {
-        --paper-radio-button-checked-color: red;
-        --paper-radio-button-checked-ink-color: red;
+      body {
```
Comment (morethanreal) [inline demo/index.html]: Move this to demo.css for sharing?

File: demo/index.html [hunk 2]
```diff
@@ -20,49 +20,97 @@
...
+        background-color: var(--paper-grey-50);
       }
 
-      paper-radio-button {
-        display: block;
+      .demo-section {
```
Comment (morethanreal) [inline demo/index.html]: Move this to demo.css for sharing?

**Pull request merged**